docs/essentials/controls.md

```diff
@@ -188,6 +188,29 @@ And here's what the resulting UI looks like:
 
 ![Controls addon expanded](./addon-controls-expanded.png)
 
+## Hide controls for certain fields on a particular story
+
+The `argTypes` annotation can be used to hide controls for a particular row, or even hide rows.
+
+Suppose you have a `Button` component with `borderWidth` and `label` properties (auto-generated or otherwise) and you want to hide the `borderWidth` row completely and disable controls for the `label` row on a specific story. Here's how you'd do that:
```

Comment:
@novellac we have room for improvement here and we can avoid having a "carbon copy" (pardon the bad pun) from the readme.

We probably can go with:
```md

## Customize stories with argTypes

Let's assume we have a `Button` component that contains the following properties, `borderWidth` and `label` and we need to create a story that hides the `borderWidth` and disable any controls for the `label`.  

How could we implement such a story? 

We can use the `argTypes` annotation in our stories to customize how they will be rendered in the UI:
```

Probably something like this.

Let me know if you're ok with the wording or we can always fine tune it.

Reply:
That's probably fine for now, but! If you're ok with improvements (famous last words):
- I'd like to add an additional bit about how to disable controls for all stories in the `export default` area (I think this is more explicit).
- Since hiding the entire `borderWidth` row in the example is done with `table.disable`, I'd like to add a line in the table under the [Customizing](https://storybook.js.org/docs/react/writing-docs/doc-blocks#customizing) section on the [Doc Blocks](https://storybook.js.org/docs/react/writing-docs/doc-blocks) page (I can make a PR for this). We could then leave off that part of the example and simplify the snippet to just address the disabled controls.
- I'd like to have the heading specify that this section addresses how to hide controls, since (hopefully) the section would just address hiding controls.

Reply:
@novellac no need for the famous last words, i'm more than ok with improvements to the documentation if they add add value to it and help out not only you but the rest of the community :)

Regarding your list:

Looking at the the pull request and the current state of the documentation, factoring in the first item, we could do better, we can break this into smaller parts, go from component to story. This will improve the documentation and help out the community understand that how they can use the annotation more effectively and that they are not strictly bound to use it in a story. 

Regarding your second item, i'm ok with adding a entry to the table and we could probably link it to this page and we present the example. 

Regarding your third item, i went with that phrasing based of the fact that in future releases the behavior can change and probably more options will be added. So it turned out a bit more agnostic.

Let me know if you're ok with this and we can make the adjustments and merge it :)